src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,4 +11,10 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
```

Comment:
why don't u delete the old commands ah

Reply:
so that nich and jordan can refer to it when implementing the commands

Reply:
for tags we reusing it